delta/plugins/composite-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/ValidateCompositeView.scala

```diff
@@ -45,7 +45,7 @@ object ValidateCompositeView {
       aclCheck.authorizeForOr(cpSource.project, events.read, cpSource.identities)(CrossProjectSourceForbidden(cpSource))
 
     def validateProject(cpSource: CrossProjectSource) =
-      projects.fetch(cpSource.project).mapError(_ => CrossProjectSourceProjectNotFound(cpSource)).void
+      projects.fetch(cpSource.project).adaptError(_ => CrossProjectSourceProjectNotFound(cpSource)).void
```

Comment:
feels like there should be an `asError` or something, analogous to `map` and `as`

Reply:
(there's not AFAIK, just complaining 😂 )